tools/cockpit.spec

```diff
@@ -148,7 +148,7 @@ make -j4 check
 
 %install
 make install DESTDIR=%{buildroot}
-make install-test-assets DESTDIR=%{buildroot}
+make install-tests DESTDIR=%{buildroot}
```

Comment:
Is it really necessary to install into the  build root and then having to explicitly enumerate the files in %files? Is it possible to say `DESTDIR=%(package_dir_of_cockpit-tests)` somehow?